cdap-ui/graphql/resolvers/applicationResolvers.js

```diff
@@ -28,6 +28,7 @@ const applicationsResolver = {
     applications: async (parent, args, context) => {
       const namespace = args.namespace;
       const options = resolversCommon.getGETRequestOptions();
+      options.headers.Authorization = context.auth;
```

Comment:
can't always assume authorization exist. Please fix this in all of the resolvers

Reply:
Done